### ISSUE TITLE
[VT-32] Scroll through history

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#54]
_Scott Kostyshak on 2013-05-03T20:35:30Z says:_ 

 Currently, if I do ":" and then <UP>, the prompt changes to "Annotate:". Same with "c" and then <UP>.

It would be nice if these scrolled through the previous commands.

If that's too difficult, I would suggest having <UP> be an escape in this context because "Annotate" does not seem intuitive here.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-05-03T20:35:30Z
Modified: 2014-06-27T14:07:18Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2014-06-08T20:09:03Z says:_ 

 duplicate?

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2014-06-09T00:54:46Z says:_ 

 Yes, thanks for catching this David. I've fixed the "Annotate:" appearing. Now <UP> and <DOWN> have no effect. But the main feature, command history, is still not implemented.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2014-06-27T14:07:18Z says:_ 

 Implemented at 716d51d8.
